contrib/database/sql/sql.go

```diff
@@ -32,6 +33,7 @@ const componentName = "database/sql"
 
 func init() {
 	telemetry.LoadIntegration(componentName)
+	tracer.MarkIntegrationImported(componentName)
```

Comment:
Is this one special?

Reply:
jk it's because the component name matches. ignore this

Reply:
Yeah I think this one (as well as a few other components) are such that `componentName` is the same as the import path for the library, so I decided to leave it as `componentName` instead of creating a new string.